examples/siro_sandbox/configs/fetch.yaml

```diff
@@ -249,8 +249,8 @@ habitat_baselines:
             nav_to_robot:
               skill_name: "NavSkillPolicy"
               obs_skill_inputs: ["goal_to_agent_gps_compass"]
-              load_ckpt_file: "data/models/point_nav_ckpt.2545.pth"
-              max_skill_steps: 750
+              load_ckpt_file: "data/models/social_nav_ckpt.1297.pth"
+              max_skill_steps: 10000000000
```

Comment:
This feels odd. Is there a better way to deactivate the step cap?

Reply:
Thanks! done!

Reply:
Thanks, done!